maisim/maisim.Game/Graphics/UserInterface/MainMenuButton.cs

```diff
@@ -1,106 +1,119 @@
+                CornerRadius = 30,
+                BorderThickness = 5,
+                BorderColour = Color4.White,
+                Children = new Drawable[]
+                {
+                    new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Anchor = Anchor.CentreRight,
+                        Origin = Anchor.CentreRight,
+                        Colour = buttonColor
+                    },
+                    new GridContainer
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        ColumnDimensions = new[]
+                        {
+                            new Dimension(GridSizeMode.Absolute, 100),
+                            new Dimension(GridSizeMode.Absolute, 400)
+                        },
+                        Content = new[]
+                        {
+                            new Drawable[]
+                            {
+                                new Container
+                                {
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    Size = new Vector2(100, 70),
+                                    Children = new Drawable[]
+                                    {
+                                        new SpriteIcon
+                                        {
+                                            Anchor = Anchor.Centre,
+                                            Origin = Anchor.Centre,
+                                            Icon = buttonIcon,
+                                            Size = new Vector2(30),
+                                            Colour = Color4Extensions.FromHex("ffffff")
+                                        }
+                                    }
+                                },new Container
+                                {
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    Size = new Vector2(400, 70),
+                                    Child = new MaisimSpriteText()
+                                    {
+                                        Anchor = Anchor.CentreLeft,
+                                        Origin = Anchor.CentreLeft,
+                                        Text = buttonText,
+                                        Font = MaisimFont.GetFont(size: 35f)
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            drawableHoverSample = new DrawableSample(sampleStore.Get("hover.wav"));
+            drawableClickSample = new DrawableSample(sampleStore.Get("click.wav"));
+        }
+
+        protected override bool OnHover(HoverEvent e)

```

Comment:
While there's an audio feedback, there's currently no visual feedback upon hovering a button which is quite confusing (at least to me). Darkening the button color upon hover might suffice to indicate that it is currently selected.

Reply:
I added `OnHover` and `OnMouseDown` so the button will darker on hover and click. Maybe we can adjust this later but this is looking good for now.